[IMP] amazon_connector: new auth flow + new marketplace setup (#728)
odoo/documentation-user: sales/amazon_connector/setup.rst

Configure Amazon Connector in Odoo
==================================

> 

Reviewer: ```suggestion

.. _amazon/generate_auth_token:

```

The Amazon Connector uses an Authorization Token that allows the Odoo Amazon
platform to fetch data from your Amazon Seller Central account. This token can
be obtained directly through Seller Central in a few clicks and will then need
to be set up in your Odoo database configuration.

Generate an Authorization Token in Seller Central
=================================================

Connect to your Seller Central account for the marketplace you initially signed
up (e.g. if you created your account on Amazon Germany, go to
`Amazon Seller Central for Germany <https://sellercentral.amazon.de>`_) with an administrator account.

.. note::
  You might need to use the *main* (or first) administrator account and not one
  added subsequentely.

In the main menu, select :menuselection:`Apps & Services --> Manage Your Apps`;
in the page that displays, click on the ``Authorize new developper`` button:

.. image:: ./media/seller_central_apps.png
  :align: center

Fill in the form depending on your Marketplace:

.. image:: ./media/seller_central_form.png
  :align: center

> For Canada, Mexico or the United States, use these values:

Reviewer: ```suggestion
If your seller account is registered in the **North America** region, use these values:
```
This is more \*robust\* imho. Some sellers could have created their account solely for the Brazil or Netherlands marketplaces. Even if we don't officially support them (but still a bit), this should be less confusing.
BTW, I looked for an official mention of the account's region on Seller Central but couldn't find any. Still, [the doc calls them as such](http://docs.developer.amazonservices.com/en_US/dev_guide/DG_Endpoints.html).

- Developer's Name: ``Odoo S.A.``

> - Developer ID: ``586127723692``

Author: @AntoineVDV plz confirm it's correct

Reviewer: I hereby confirm the correctness of this information.

Author: ![382909-20.jpeg](https://user-images.githubusercontent.com/9530767/88529591-7df4e400-d000-11ea-824a-90cfe4361b96.jpeg)

> For Germany, Spain, France or Italy, use these values:

Reviewer: same (co.uk is missing btw)

- Developer's Name: ``Odoo S.A.``

> - Developer ID: ``579095187166``

Author: @AntoineVDV plz confirm it's correct

Reviewer: correct

Amazon will then inform you that by submitting the form, you are giving access
to your Seller Central Information to Odoo S.A.

.. note::
    Odoo S.A. is unable to access your Amazon account's information without the
    Authorization Token which is stored in your Odoo database - we do not store
    these tokens on our platform directly and are therefore unable to access
    your account's information outside of the Amazon Connector normal flows.

Register your Amazon account in Odoo
====================================

.. _amazon/setup:

To register your seller account in Odoo, navigate to :menuselection:`Sales --> Configuration
--> Settings --> Connectors --> Amazon Sync --> Amazon Accounts` and click on **CREATE**.

The **Seller ID** can be found in Seller Central under the link **Your Merchant Token** on the
**Seller Account Information** page. The **Authorization Token** is the one you generated in the

> previous step.

Reviewer: ```suggestion
:ref:`previous step <amazon/generate_auth_token>`.
```
